benchmark_utils/augmented_dataset.py

```diff
@@ -27,6 +28,8 @@ def name(self):
         pass
 
     def run(self, n_iter):
+        n_samples = [0.1, 0.25, 0.5, 0.7, 1, 2, 5, 7, 10, 20]
```

Comment:
this should only be defined in l.42 as this is the only place it is used.
As a rule of thumb, avoid defining specific part of the function in the global scope. Defining them close to where they are used make it easier to read the code.